### ISSUE TITLE
html/static/components/impress.js/js/impress.js missing on build

### ISSUE THREAD

**Issue by bmsleight** [gawel/impress#5]
using latest version via:
$ pip install http://github.com/gawel/impress/tarball/master

$ impress
/home/bms/impress/docs/index.rst:None: WARNING: nonlocal image URI found: https://upload.wikimedia.org/wikipedia/commons/0/0a/Python.svg
$ ls html/static/components/impress.js/js/impress.js 
ls: cannot access html/static/components/impress.js/js/impress.js: No such file or directory

Had to use this command to resolve;
$wget http://gawel.github.io/impress/static/components/impress.js/js/impress.js -o impress.js; mkdir -p html/static/components/impress.js/js/ ;  cp impress.js html/static/components/impress.js/js/ 

I still love this project :)

**Comment by gawel** [comment]
Yeah you cant install from github's tarball. this is because external resources are fetched with bower:

https://github.com/gawel/impress/blob/master/bower.json 

Don't know if I can disable tarballs. Btw I wont fix this. I'd like to avoid having impress.js code in the repo.

Simple solution: use the release from pypi
